db/migrate/20210125215911_create_roles.rb

```diff
@@ -0,0 +1,8 @@
+class CreateRoles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :roles do |t|
+      t.string :name,        null: false, limit: 50
+      t.text   :description, limit: 280
+    end
+  end
+end
```

Comment:
I think it's much easier and better to create roles by adding a simple enum on the User (or a string field, alternatively). No need to create a separate table for it